files/fr/web/svg/attribute/cx/index.md

```diff
@@ -42,17 +42,7 @@ Pour un élément {{SVGElement('circle')}}, `cx` définit la coordonnée x de so
   <tbody>
     <tr>
       <th scope="row">Valeur</th>
-      <td>
-        <strong
-          ><a href="/docs/Web/SVG/Content_type#Length">&#x3C;length></a></strong
-        >
-        |
-        <strong
-          ><a href="/docs/Web/SVG/Content_type#Percentage"
-            >&#x3C;percentage></a
-          ></strong
-        >
-      </td>
+      <td>{{cssxref("length-percentage")}}</td>
```

Comment:
The formatting is subjective, I was thinking that this may be slightly easier to read

```suggestion
      <td><a href="/fr/docs/Web/CSS/length-percentage">
        <code>&lt;length-percentage&gt;</code>
      </a></td>
```